helpers/parse_args.go

```diff
@@ -252,9 +252,6 @@ func OverrideLocation(available []string) {
 
 // ReadEnvFile reads the .env file and loads its environment variables.
 func ReadEnvFile() error {
-	err := gotenv.Load() // to allow use of .env file
-	if err != nil && !strings.HasPrefix(err.Error(), "open .env:") {
-		return err
-	}
-	return nil
+	err := envy.Load("../.env") // to allow use of .env file
```

Comment:
So this looks at the parent of the working directory?

I'd setup a hierarchy of potential places to look. 
[working directory], [$GOPATH/src/github.com/Azure/azure-sdk-for-go-samples]